client/asset/xmr/version.go

```diff
@@ -89,9 +87,5 @@ func (m *moneroVersion) compare(other *moneroVersion) int {
 
 func (m *moneroVersion) valid() bool {
 	c := m.compare(minimumVersion)
-	if c < 0 {
-		return false
-	}
-	c = m.compare(recommendedVersion)
-	return c <= 0
+	return c >= 0
 }
```

Comment:
There is an upper limit, if any of the other values, min, max, or sys were higher it would not be compatible. Just the patch doesn't really matter.